analysis/pseudobulk-bulk-prediction/model-notebooks/SCPCP000001_bulk-pseudobulk.Rmd

```diff
@@ -0,0 +1,141 @@
+---
+title: "Predict bulk from pseudobulk: SCPCP000001"
+author: Stephanie J. Spielman
+date: "`r Sys.Date()`"
+output:
+  html_notebook:
+    toc: true
+    toc_depth: 3
+    code_folding: hide
+---
+
+_Introduction coming later._
+
+## Setup
+
+```{r setup}
+renv::load()
+
+library(ggplot2)
+theme_set(theme_bw())
+```
+
+
+
+```{r}
+project_id <- "SCPCP000001"
+
+# As identified in https://github.com/AlexsLemonade/scpca-paper-figures/issues/135
+exclude_samples <- c("SCPCS000003")
+```
+
+
+### Paths
+
+```{r paths}
+analysis_dir <- here::here("analysis", "pseudobulk-bulk-prediction")
+data_dir <- file.path(analysis_dir, "data")
+metadata_dir <- file.path(rprojroot::find_root(rprojroot::is_renv_project), "s3_files") 
```

Comment:
Is this different from:
```suggestion
metadata_dir <- here::here("s3_files") 
```

Reply:
It is not, as (cc stephanie) this is not an openscpca module.